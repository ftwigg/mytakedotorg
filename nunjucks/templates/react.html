{% extends "reactCore.html" %}

{% block bundle %}
	<!-- Main -->
	<script src="/app.bundle.js"></script>
{% endblock %}
