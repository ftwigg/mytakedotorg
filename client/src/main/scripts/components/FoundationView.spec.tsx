import * as React from "react";
import * as renderer from "react-test-renderer";
import {} from "jest";
import FoundationView from "./FoundationView";

jest.mock("./TimelineView", () => ({
  default: "TimelineView"
}));

test("With hash URL", () => {
  const hashURL =
    "#bill-of-rights&/samples/does-a-law-mean-what-it-says-or-what-it-meant&294&368&283&439&479.734375";
  const tree = renderer.create(<FoundationView hashUrl={hashURL} />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Without hash URL", () => {
  const hashURL = "";
  const tree = renderer.create(<FoundationView hashUrl={hashURL} />).toJSON();
  expect(tree).toMatchSnapshot();
});
