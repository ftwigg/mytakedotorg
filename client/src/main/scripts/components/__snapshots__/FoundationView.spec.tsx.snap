// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With hash URL 1`] = `
<div
  className="foundation"
>
  <TimelineView
    hashUrl="#bill-of-rights&/samples/does-a-law-mean-what-it-says-or-what-it-meant&294&368&283&439&479.734375"
  />
</div>
`;

exports[`Without hash URL 1`] = `
<div
  className="foundation"
>
  <div>
    <div
      className="foundation__inner-container"
    >
      <h1
        className="foundation__heading"
      >
        Foundation of Facts
      </h1>
      <p
        className="timeline__instructions"
      >
        Explore Facts in the timeline below.
      </p>
    </div>
    <TimelineView />
  </div>
</div>
`;
